### PR TITLE
[DOCS] [8.1] Add references to Maps permissions, update map screenshot

### LINKED ISSUE #1604: [DOCS] Network map can be expanded or hidden
## Description 

Addresses UX changes that occurred as part of the fix in https://github.com/elastic/kibana/pull/123336. 

If a user's role has `ALL` or `READ` privileges to Maps, they'll have access to the button that allows them to expand or hide the Network map on the Security page. 

maps

maps-expand-hide

## Notes

- In the [Network page overview](https://www.elastic.co/guide/en/security/current/network-page-overview.html) topic, we'll need to document which role privs are needed to access the network map and update the screenshot at the start of the topic (`network-ui.png`). The priv docs can probably go before the note, after the screenshot. 
- In the [Configure network map data](https://www.elastic.co/guide/en/security/master/conf-map-ui.html) topic, we only need to document required role privs. Since this topic is divided into sections, I recommend adding a new section and linking to it in the bulleted list at the start of the topic.

### PR DESCRIPTION
**Pull request by benironside**
Fixes #1604 

The section that I added in Configure network map data (net-map-req.asciidoc) is currently called Permissions required, but I don't love this name. I would have called it Prerequisites at Cmd. Any suggestions?

Previews:
[Network page overview](https://security-docs_1649.docs-preview.app.elstc.co/guide/en/security/master/network-page-overview.html)
[Configure network map data](https://github.com/elastic/security-docs/edit/main/docs/getting-started/net-map-req.asciidoc)

### CODE REVIEW

File: docs/getting-started/net-map-req.asciidoc [hunk 1]
```diff
@@ -11,6 +11,11 @@ Depending on your {kib} setup, to display and interact with data on the
 NOTE: To see source and destination connections lines on the map, you must
 configure `source.geo` and `destination.geo` ECS fields for your indices.
 
+[float]
+[[prereq-perms]]
+=== Permissions required
+In order to view the map, you need at least "Read" privileges for `Maps`. To configure it, you need "All" privileges. Maps privilege settings are under *Kibana privileges* -> *Analytics* -> *Maps*.
```
Comment (joepeeples) [inline docs/getting-started/net-map-req.asciidoc:17]: Per other pages concerning role privs (like [this](https://www.elastic.co/guide/en/security/8.0/detections-permissions-section.html)), backticks for the privilege options:
```suggestion
In order to view the map, you need at least `Read` privileges for `Maps`. To configure it, you need `All` privileges. Maps privilege settings are under *Kibana privileges* -> *Analytics* -> *Maps*.
```

File: docs/getting-started/network-page-overview.asciidoc [hunk 2]
```diff
@@ -9,7 +9,7 @@ further investigation.
 [role="screenshot"]
 image::images/network-ui.png[]
 
-NOTE: For information on how to set up the interactive map, see <<conf-map-ui>>.
+NOTE: To access the interactive map, you need either "Read" or "All" privileges for `Maps` (*Kibana Privileges* -> *Analytics* -> *Maps*). To learn more about map setup refer to <<conf-map-ui>>.
```
Comment (joepeeples) [inline docs/getting-started/network-page-overview.asciidoc:12]: Character formatting, and missing comma:
```suggestion
NOTE: To access the interactive map, you need either `Read` or `All` privileges for `Maps` (*Kibana Privileges* -> *Analytics* -> *Maps*). To learn more about map setup, refer to <<conf-map-ui>>.
```

**Review by joepeeples — COMMENTED** [review]
Just a couple formatting edits. 

Also, could you crop the image to remove the top bars and nav sidebar, similar to the original? Thanks!

**Review by jmikell821 — APPROVED** [review]
OK to merge - thanks!

**Comment by nastasha-solomon** [comment]
Had the same input as @joepeeples , thanks!

**Review by nastasha-solomon — APPROVED** [review]

**Pull request merged**